ONSdigital/census-rh-ui
# CR-1792 estabType check

Implements a check for estabType before use. Also added new unit tests to validate change

Signed-off-by: Paul-Joel <paul.joel@ons.gov.uk>

# Motivation and Context
Fix to the NoneType error caused by missing estabType

# What has changed
Code now checks estabType exists before trying to check it's value
Added new unit tests and test data to confirm

No Cucumber changes required

# How to test?
Enter a UAC for a case without an estabType, and ensure it doesn't cause an error in the UI
Enter a transient UAC and confirm that it still goes to the transient pages

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1792
